functions/Remove-DbaDbSequence.ps1

```diff
@@ -1,14 +1,16 @@
 function Remove-DbaDbSequence {
     <#
     .SYNOPSIS
-        Removes a sequence.
+        Removes sequences.
 
     .DESCRIPTION
-        Removes a sequence in the database(s) specified.
+        Removes the sequences that have passed through the pipeline.
+
+        If not used with a pipeline, Get-DbaDbSequence will be executed with the parameters provided
+        and the returned sequences will be removed.
 
     .PARAMETER SqlInstance
-        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function
-        to be executed against multiple SQL Server instances.
+        The target SQL Server instance(s).
```

Comment:
This should be reverted as the description of this parameter is used across all commands.

Reply:
OK, the term "The target SQL Server instance or instances." is widely used, will change that.
But "This can be a collection and receive pipeline input ..." is not correct here as the the command only allows pipeline input of type Microsoft.SqlServer.Management.Smo.Sequence. 
I know that you would like to have the commands allow different objects to be piped in, I took part in that disussion: https://github.com/sqlcollaborative/dbatools/pull/7320 - Can we for now only accept input like it is coded rigth now and maybe later change all the Remove-DbaDb* commands to work the same way?